### ISSUE TITLE
Train and evaluate on VOC

### ISSUE THREAD

**Issue by huohuoR** [CSAILVision/semantic-segmentation-pytorch#47]
Hi, all
 
I am trying to train the network on VOC dataset and I did:
1. modify the dataset into ADE format
2. change the "num_class" to 21
3. segm += 1: add the ground truth label by 1 both for training and evaluation. (because in your code class 0 didn't count in loss and mIoU)

Am I doing this right?

Any help is appreciated.

**Comment by hangzhaomit** [comment]
@huohuoR yes, I think you are on the right track

**Comment by huohuoR** [comment]
@hangzhaomit  Thanks for reply. Looking forward to you update.

**Comment by hangzhaomit** [comment]
@huohuoR currently we do not have enough bandwidth to include other datasets, instead we are working to provide better models.

**Comment by littlebelly** [comment]
@huohuoR 
Have you train these model on voc dataset? how about the performance? and what's your meaning of 'modify the dataset into ADE format? could you share your code with me? Thanks a los!
